Dockerfile

```diff
@@ -144,7 +159,17 @@ RUN pip install pybids==0.5.1
 ENV PYTHONPATH=""
 
 COPY run.py /run.py
-RUN chmod +x /run.py
+RUN chmod 555 /run.py
 
 COPY version /version
+COPY IntendedFor.py /IntendedFor.py
+COPY modified_files/fsl_sub /usr/lib/fsl/5.0/fsl_sub
+COPY modified_files/settings.sh /opt/fix/settings.sh
+COPY modified_files/360CortSurf_19Vol_parcel.dlabel.nii /360CortSurf_19Vol_parcel.dlabel.nii
+COPY modified_files/PostFix.sh /opt/HCP-Pipelines/PostFix/PostFix.sh
+COPY modified_files/run_prepareICAs.sh /opt/HCP-Pipelines/PostFix/Compiled_prepareICAs/distrib/run_prepareICAs.sh
+COPY modified_files/RestingStateStats.sh /opt/HCP-Pipelines/RestingStateStats/RestingStateStats.sh
+COPY modified_files/run_RestingStateStats.sh /opt/HCP-Pipelines/RestingStateStats/Compiled_RestingStateStats/distrib/run_RestingStateStats.sh
```

Comment:
I would like to avoid departing from the original HCP Pipelines as much as possible. Why those files need to be replaced?

Reply:
Generally I absolutely agree with you, however, I was required to make mods to the RestingStateStats and PostFix scripts because the paths to the matlab compiler runtime are hardcoded to paths that fall within WashU's server systems.